code/game/machinery/machinery.dm

```diff
@@ -119,17 +119,11 @@ Class Procs:
 
 	var/speed_process = FALSE			//If false, SSmachines. If true, SSfastprocess.
 
-<<<<<<< HEAD
 	blocks_emissive = EMISSIVE_BLOCK_GENERIC
 
-/obj/machinery/New(l, d=0)
-	..()
-	if(isnum(d))
-=======
 /obj/machinery/Initialize(var/ml, d=0)
 	. = ..()
 	if(d)

```

Comment:
It being isnum(d) seems like it was a vorestation change. Not sure why, but probably should keep it just in case?

Reply:
Yes, this was a vorestation change to fix a bug. Should probably remain as such.

Reply:
done

should probably be upstreamed if it's a serious bug. I've flagged it as an edited line for now.